pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -8,12 +8,14 @@ import { escapeHtml } from '@shell/utils/string';
 import cronstrue from 'cronstrue';
 import { isValidCron } from 'cron-validator';
 import { debounce } from 'lodash';
-
+import VeeTokenValidation from 'components/VeeValidation.vue';
```

Comment:
First thing, this is the correct path:
```ts
import VeeTokenValidation from '@shell/components/VeeValidation.vue';
```

Second, you cannot reference something from the `shell` is you are in `pkg`.